src/main/drivers/timer_hal.c

```diff
@@ -114,6 +114,7 @@ static uint8_t lookupTimerIndex(const TIM_TypeDef *tim)
 #if USED_TIMERS & TIM_N(8)
         _CASE(8);
 #endif
+#if !defined(STM32H7)
```

Comment:
isn't it possible to simply not set bits 9,10,11 in USED_TIMERS ?

Reply:
What do you mean? We turn off bits 9, 10 and 11 after the inclusion of `target.h` (may be in `common_post.h`)?
```
#define USED_TIMERS_HWCONSTRAINED    (USED_TIMERS & (~(TIM_N(9)|TIM_N(10)|TIM_N(11)))
```

Reply:
Why does even target.h enable timers that are nonexistent on that architecture?

Maybe define TIMERS_AVAILABLE macro for target.h use? Or, with unified targets, change the logic so that target.h will only disable timers if necessary ...

Reply:
Okay ... I'll just let it be an undisturbed straight `case`(s) here.
Unavailable timers will be flagged up by a compiler.
In unified target case, it will be flagged up by `timerGetByTagAndIndex`.